src/route/user.route.js

```diff
@@ -0,0 +1,42 @@
+import userController from '../controller/user.controller.js'
+import express from 'express'
+import schema from './user.post.schema'
+
+const userRouter = express.Router();
+
+userRouter.post('/add', validateSchema(schema), userController.addUser);
+userRouter.get('/all', userController.getAllUsers);
+userRouter.get('/:id', userController.getUserById);
+userRouter.get('/:login/:limit', userController.getAutoSuggestUsers);
+userRouter.put('/:id', validateSchema(schema), userController.updateUser);
+userRouter.delete('/:id', userController.deleteUser);
+
+
+function errorResponse(schemaErrors) {
+    const errors = schemaErrors.map((error) => {
```

Comment:
in js you can omit `return` keyword if there is no other logic in function and you want just to return something. In this case you should return result in parentheses (...).
In this example: `schemaErrors.map((error) => ({
    path: error.path,
    message: error.message
  }))`